ROS buildfarm failure for 2dnav and bringup pkgs

ros-indigo-spur-2dnav: http://jenkins.ros.org/job/ros-indigo-spur-2dnav_binarydeb_trusty_amd64/2/
ros-indigo-spur-bringup: http://jenkins.ros.org/job/ros-indigo-spur-bringup_binarydeb_trusty_i386/2/

```
CMake Error at /opt/ros/indigo/share/catkin/cmake/catkinConfig.cmake:75 (find_package):
  Could not find a package configuration file provided by "urg_node" with any
  of the following names:

    urg_nodeConfig.cmake
    urg_node-config.cmake

  Add the installation prefix of "urg_node" to CMAKE_PREFIX_PATH or set
  "urg_node_DIR" to a directory containing one of the above files.  If
  "urg_node" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  CMakeLists.txt:7 (find_package)
```
I think `urg_node` doesn't need to be `find_package`ed since it's a `run_depend` only.

--- a/spur_2dnav/CMakeLists.txt
+++ b/spur_2dnav/CMakeLists.txt
@@ -4,14 +4,7 @@ project(spur_2dnav)
 ## Find catkin macros and libraries
 ## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
 ## is used, also find other catkin packages
-find_package(catkin REQUIRED COMPONENTS
-  amcl
-  dwa_local_planner
-  gmapping
-  map_server
-  move_base
-  tf
-)
+find_package(catkin REQUIRED)
 
 ## System dependencies are found with CMake's conventions
 # find_package(Boost REQUIRED COMPONENTS system)

--- a/spur_bringup/CMakeLists.txt
+++ b/spur_bringup/CMakeLists.txt
@@ -4,10 +4,7 @@ project(spur_bringup)
 ## Find catkin macros and libraries
 ## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
 ## is used, also find other catkin packages
-find_package(catkin REQUIRED COMPONENTS
-  urg_node
-  rostest
-)
+find_package(catkin REQUIRED rostest)
 
 ## System dependencies are found with CMake's conventions
 # find_package(Boost REQUIRED COMPONENTS system)

